README.md

```diff
@@ -55,7 +55,7 @@ Start an anger instance against a given target.
     * `connections`: The number of connections to maintain to server. Each is an individual nes client.
     * `senders`: The number of connections that should be senders.
     * `responses`: The number of overall expected responses to be recieved during this run.
-    * `auth`: An `Object` passed to the nes client for authentication. [reference.][nes-auth]
+    * `auth`: A `Function` passed to the nes client for authentication. This is passed the `client` and `index` of that client as params. Must return the auth object options. [reference.][nes-auth]
```

Comment:
In the auth function, I was mainly interested to know which index the client is and whether or not it is one of the sender clients. I didn't need the full client object. 

I know I could technically figure out as a user whether or not it's a sender based on the index, but that feels like making too many assumptions about the internals staying the same. What do you think?